modules/customize-certificates-manually-rotate-service-ca.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-signing-certificates.adoc
+
+[id="manually-rotate-service-ca-{context}"]
+= Manually rotate the service CA certificate
+
+The service CA is valid for one year after {product-title} is 
+installed. If you have a cluster that has been running
+for one year without upgrades, then you will need to manually
+refresh the service CA.
+
+This process is automatically performed during an upgrade,
+and should only be followed for clusters that do not perform
+upgrades.
+
+.Prerequisites
+
+* You must be logged in as a cluster admin.
+
+.Procedure
+
+. View the expiration date of the current service CA certificate by
+using the following command. If this certificate expires
+before a scheduled upgrade, then proceed through the following steps.
+Otherwise, there is no need to perform the rotation at this time.
++
+----
+$ oc get secrets/signing-key -n openshift-service-ca -o template='{{index .data "tls.crt"}}' | base64 -d | openssl x509 -noout -enddate
+----
+
+. Manually rotate the service CA.
++
+----
+$ oc delete secret/signing-key -n openshift-service-ca
+----
+
+. Restart all Pods. This refreshes all Pods so that they receive the newly
+created service certificates created by the recently rotated service CA. 
+This command ensures that all services use the updated certificates.
++
+----
+$ for I in $(oc get ns -o jsonpath='{range .items[*]} {.metadata.name}{"\n"} {end}'); do oc delete pods --all -n $I; sleep 1; done
+----
++
+[WARNING]
+====
+This command will cause a service interruption, as it goes through and
+deletes every running pod in every namespace.
+====
```

Comment:
Is there another logical command to put here? Like watching the Pods come back online or checking to see if they have the right cert?

Reply:
The command tells you as each pod is restarted. I'm going to leave this as is for now, as if you've executed the steps in order then nothing else is required.